cmd/checkIfStepActive.go

```diff
@@ -152,11 +154,19 @@ func addCheckStepActiveFlags(cmd *cobra.Command) {
 
 func initializeConfig(pConfig *config.Config) (*config.Config, error) {
 	projectConfigFile := getProjectConfigFile(GeneralConfig.CustomConfig)
-	customConfig, err := checkStepActiveOptions.openFile(projectConfigFile, GeneralConfig.GitHubAccessTokens)
-	if err != nil {
-		return nil, errors.Wrapf(err, "config: open configuration file '%v' failed", projectConfigFile)
+	var customConfig io.ReadCloser
+	var err error
+	//accept that config file cannot be loaded as its not mandatory here
+	if exists, err := checkStepActiveOptions.fileExists(projectConfigFile); exists {
```

Comment:
error case not handled, no idea if this is relevant or will result in exists = false

Reply:
This is referring to [this line](https://github.com/SAP/jenkins-library/blob/72e257e83ef2467a7764679ef4c76a2b7d535bb4/pkg/piperutils/fileUtils.go#L64) ,it returns false if file doesn't exists. Also there is a case where it returns nil